cmd/sandbox/start.go

```diff
@@ -110,14 +135,23 @@ func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bu
 	}
 
 	if len(sandboxConfig.DefaultConfig.Version) > 0 {
-		if err := downloadFlyteManifest(sandboxConfig.DefaultConfig.Version); err != nil {
-			return nil, err
-		}
-		vol, err := mountVolume(FlyteManifest, GeneratedManifest)
+		isGreater, err := util.IsVersionGreaterThan(sandboxConfig.DefaultConfig.Version, flyteMinimumVersionSupported)
 		if err != nil {
 			return nil, err
 		}
-		volumes = append(volumes, *vol)
+		if !isGreater {
+			logger.Infof(ctx, "version flag only supported after with flyte %s+ release", flyteMinimumVersionSupported)
```

Comment:
Should we abort instead of continuing with pulling the image and just logging it . The action is just to log ? what does that tell to the user to do